plugins/labhub.py

```diff
@@ -279,12 +279,14 @@ def difficulty_level(user, iss):
                 2. The user belongs to developers or maintainers team as well as
                    newcomers team.
                 False if
-                1. A newcomer asks for assignment to an issue that has no
+                1. A member asks for assignment to an issue that has 
+                   initiatives/gci label.
+                2. A newcomer asks for assignment to an issue that has no
                    difficulty label.
-                2. A newcomer asks for assignment to an issue with difficulty
+                3. A newcomer asks for assignment to an issue with difficulty
                    higher than low.
                 """
-                if (self.TEAMS[self.GH_ORG_NAME + ' newcomers'].is_member(user)
+                elif (self.TEAMS[self.GH_ORG_NAME + ' newcomers'].is_member(user)
                     and not (self.TEAMS[self.GH_ORG_NAME +
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpapvdlqse/plugins/labhub.py
+++ b/tmp/tmpapvdlqse/plugins/labhub.py
@@ -287,8 +287,8 @@
                    higher than low.
                 """
                 elif (self.TEAMS[self.GH_ORG_NAME + ' newcomers'].is_member(user)
-                    and not (self.TEAMS[self.GH_ORG_NAME +
-                                        ' developers'].is_member(
+                      and not (self.TEAMS[self.GH_ORG_NAME +
+                                          ' developers'].is_member(
                             user
                         ) or
                         self.TEAMS[self.GH_ORG_NAME + ' maintainers'].is_member(
```